src/Compilers/CSharp/Portable/Symbols/Source/SourceMemberContainerSymbol.cs

```diff
@@ -2992,10 +2992,12 @@ private void AddSynthesizedRecordMembersIfNecessary(MembersAndInitializersBuilde
                 addDeconstruct(ctor.Parameters, existingOrAddedMembers);
             }
 
+            var baseClone = SynthesizedRecordClone.FindValidCloneMethod(BaseTypeNoUseSiteDiagnostics);
```

Comment:
>var baseClone = SynthesizedRecordClone.FindValidCloneMethod(BaseTypeNoUseSiteDiagnostics); [](start = 12, length = 90)

This change looks unrelated to the stated goal of this PR, consider extracting it into a separate PR in order to speed up completion of this PR.